cli.ts

```diff
@@ -425,6 +430,7 @@ const initProject = (
 		mkInitialDirectories(projectDir, maxConcurrency, i18n),
 		chain(_ => exec('npm init -y 2>&1 > /dev/null', {}, false, projectDir)),
 		chain(_ => exec('taq install @taqueria/plugin-core 2>&1 > /dev/null', {}, false, projectDir)),
+		map(_ => createGitIgnoreFile(projectDir)),
```

Comment:
Shouldn't this be `chain`?

Reply:
No because `createGitIgnoreFile` is a function that returns a future